com.unity.ml-agents/Runtime/SideChannels/EnvironmentParametersChannel.cs

```diff
@@ -0,0 +1,74 @@
+using System.Collections.Generic;
+using System;
+
+namespace MLAgents.SideChannels
+{
+    /// <summary>
+    /// A side channel that manages the environment parameter values from Python. Currently
+    /// limited to parameters of type float.
+    /// </summary>
+    internal class EnvironmentParametersChannel : SideChannel
+    {
+        Dictionary<string, float> m_Parameters = new Dictionary<string, float>();
+        Dictionary<string, Action<float>> m_RegisteredActions =
```

Comment:
Put these dictionaries in EnvironmentParameters class

Reply:
We can make this a separate PR as we didn't fully discuss how the messages get passed to EnviromentParameters. (Also not an API change, so less urgent.)